frontend/app_flowy/packages/flowy_infra/lib/colorscheme/colorscheme.dart

```diff
@@ -1,6 +1,22 @@
+import 'package:flowy_infra/theme.dart';
 import 'package:flutter/material.dart';
-
```

Comment:
Please revert. A newline separates external and internal packages

Reply:
Sure